src/sentry/static/sentry/app/views/releases/list/releaseLandingCard.jsx

```diff
@@ -27,14 +27,11 @@ class ReleaseLandingCard extends React.Component {
 
   render() {
     const {card, cardsLength, step} = this.props;
-    const CardComponent = card.component;
     const finalStep = step === cardsLength - 1;
     return (
       <Container>
         <IllustrationContainer>
-          <CardComponentContainer>
-            <CardComponent />
-          </CardComponentContainer>
+          <CardComponentContainer>{card.svg}</CardComponentContainer>
```

Comment:
Could you follow-up with another PR to convert this to TS (and improve proptypes for `props.card`) since you're now a typescript GOD.